### ISSUE TITLE
Library Intergration

### ISSUE THREAD

**Issue by ShaVerZ** [123Venom/plugin.video.venom#59]
I don't know what I'm doing wrong but I can setup my Kodi Library and all looks great but new titles (shows/movies) are not getting added, the only way I can add it to the Kodi Library is by going into Venom, selecting the title and Add to Library.

As I couldn't get it working, I installed the Trakt Script and that won't update and add the titles neither.

If someone could point me to the best way to setup library integration, I'd be most grateful.

Perhaps there's a better way..... 

Many thanks, hope everyone is well.

**Comment by 123Venom** [comment]
library is updating fine here and I have one keybase user that is heavy on the library usage and not heard of any issues with his shows not getting updated when new eps air.  If you can get on keybase shoot me a PM when ya can and we can hook up for a little one on one and see if we can't resolve it.

**Comment by ShaVerZ** [comment]
Hey! I've been looking at this again today - I've given up! I've decided to use OpenMeta from your Repo instead.

I downgraded my Kodi in case that was the issue but that wasn't the problem.

The strange thing was, it seemed to be doing everything as it should upon Kodi startup and scheduling but it simply wouldn't transfer the data into the Kodi libraries. 

Anyway, i'll complete a new setup with OpenMeta, i think this option will be more suited to multiple setups, across various devices.

Just like to say thanks for your support and hard work - much appreciated. Stay Safe.
